Sources/MapboxMaps/Ornaments/Compass/MapboxCompassOrnamentView.swift

```diff
@@ -55,7 +57,11 @@ internal class MapboxCompassOrnamentView: UIButton {
                                               comment: "Accessibility hint")
 
         if let image = createCompassImage() {
+            let bounds = CGRect(origin: .zero, size: image.size)
+            compassContainerView.frame = bounds
             setImage(image, for: .normal)
+            compassContainerView.alpha = visibility == .visible ? 1 : 0
```

Comment:
does this mean `adaptive` is not visible? Does that clash with any docs?

Reply:
This is adapted from the [previous way](https://github.com/mapbox/mapbox-maps-ios/blob/main/Sources/MapboxMaps/Ornaments/Compass/MapboxCompassOrnamentView.swift#L44) we set the alpha for the compass view, just applied to the compass container view, so behavior shouldn't change.